2_base_and_sensor.ipynb

```diff
@@ -300,7 +547,26 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Please check the data you are interested in from \"Displays\" panel on the left screen. The meaning of each item is as follows:\n",
```

Comment:
"Please check the data you are interested in using the \"Displays\" panel on the left side of the screen. The meaning of each item is as follows:\n",

OK, I replaced "from" with "using the" and replaced "on the left screen" with "on the left side of the screen".